README.md

```diff
@@ -46,6 +46,68 @@ UnDeploy the controller to the cluster:
 make undeploy
 ```
 
+### Building your own bundle, index images
```

Comment:
We might want to mention the `IMAGE_TAG_BASE` env var here, as these commands by default will try to eventually push to `quay.io/openstack-k8s-operators`

Reply:
But wait, IMAGE_TAG_BASE ?= quay.io/$(USER)/openstack-operator right? When I run them locally they push to quay.io/dprince

Reply:
You're right.  It's just my environment that requires me to override this to get it to work.  Nevermind.